ccm_web/package.json

```diff
@@ -40,6 +40,7 @@
     "csurf": "^1.11.0",
     "express": "^4.17.1",
     "express-session": "^1.17.2",
+    "form-data": "^4.0.0",
```

Comment:
If this is indeed necessary, you also should update `package-lock.json`. There should have been some changes to it if you ran `npm install` locally within `ccm_web`.

Reply:
Yes, it's necessary, as far as I can tell.  I didn't see another way to get parameters properly formatted to send to the invitation service.

I have a couple of other additions to `package.json`, but I'm not sure whether **_they_** are necessary, which is why I've not submitted the lockfile yet.

Reply:
Still need the lockfile changes.

Reply:
Lockfile changes seem to be present now. Resolving...